.github/workflows/pr-ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [3.6, 3.7, 3.8]
+        python: [3.7, 3.8]
```

Comment:
Do we need to remove 3.6 in this PR?  Wanted to include this after the core package has its next official release

Reply:
Newer version of the pre-commit linting look to be broken in python 3.6.  Will look to backdate the versions to support 3.6